<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_date">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Day Now</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_control">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Control Panel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_today">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>391</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Today's Target</string>
   </property>
   <widget class="QPushButton" name="btn_save_del">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save or Delete</string>
    </property>
   </widget>
   <widget class="QFrame" name="fr_n1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_add1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_n1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>科目1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_add1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_1"/>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_n2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_add2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_n2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>科目2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_add2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_2"/>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_n3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_add3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_n3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>科目3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_add3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_3"/>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_warn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Warn text</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="fr_cal">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>311</width>
     <height>661</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCalendarWidget" name="cw_calendar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>312</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="fr_n3_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_n3_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>科目3</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_n3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_n2_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_n2_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>科目2</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_n2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_n1_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_n1_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>科目1</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_n1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_mdAgenda">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Modify agenda</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
